parent/pom.xml

```diff
@@ -379,14 +382,20 @@
             </dependency>
             <dependency>
                 <groupId>org.mockito</groupId>
-                <artifactId>mockito-all</artifactId>
-                <version>1.9.5</version>
+                <artifactId>mockito-core</artifactId>
+                <version>${mockito.version}</version>
                 <scope>test</scope>
             </dependency>
             <dependency>
                 <groupId>org.testng</groupId>
                 <artifactId>testng</artifactId>
-                <version>6.8.5</version>
+                <version>${testng.version}</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>junit</groupId>
+                <artifactId>junit</artifactId>
+                <version>${junit.version}</version>
```

Comment:
not sure i get your point:
```
mvn dependency:tree   > tree | grep junit
[INFO]    +- junit:junit:jar:4.13.1:test
[INFO]    +- junit:junit:jar:4.13.1:test
[INFO] |  \- junit:junit:jar:4.13.1:test
[INFO]    +- junit:junit:jar:4.13.1:test
[INFO] |  +- junit:junit:jar:4.13.1:test
[INFO]    +- junit:junit:jar:4.13.1:test
[INFO]    +- junit:junit:jar:4.13.1:test
[INFO] |  +- junit:junit:jar:4.13.1:test
[INFO]    +- junit:junit:jar:4.13.1:test
[INFO] |  \- junit:junit:jar:4.13.1:test
[INFO] |  +- junit:junit:jar:4.13.1:test
[INFO] |  +- junit:junit:jar:4.13.1:test
[INFO] |  |  |     |  |  \- junit:junit:jar:4.13.1:test
[INFO] |  +- junit:junit:jar:4.13.1:test
[INFO] |  +- junit:junit:jar:4.13.1:test
[INFO] |  +- junit:junit:jar:4.13.1:test
[INFO] |  +- junit:junit:jar:4.13.1:test
[INFO]    +- junit:junit:jar:4.13.1:test
[INFO]    +- junit:junit:jar:4.13.1:test
[INFO] |  +- junit:junit:jar:4.13.1:test
[INFO]    +- junit:junit:jar:4.13.1:test
[INFO] |  +- junit:junit:jar:4.13.1:test
[INFO] |  +- junit:junit:jar:4.13.1:test
[INFO]    +- junit:junit:jar:4.13.1:test
[INFO]    +- junit:junit:jar:4.13.1:test
[INFO]    |  \- junit:junit:jar:4.13.1:test
[INFO] |     |  \- junit:junit:jar:4.13.1:test
[INFO]    +- junit:junit:jar:4.13.1:test
[INFO] |  +- junit:junit:jar:4.13.1:test
[INFO] |  +- junit:junit:jar:4.13.1:test
[INFO]    \- junit:junit:jar:4.13.1:test
[INFO]    +- junit:junit:jar:4.13.1:test
[INFO] |  \- junit:junit:jar:4.13.1:test
[INFO] |  +- junit:junit:jar:4.13.1:test
[INFO] |  +- junit:junit:jar:4.13.1:test
```

Reply:
If you run `mvn dependency:tree` from quality/integration you will get following:
```
[INFO] +- com.bazaarvoice.curator:test:jar:1.4.1:test
[INFO] |  \- junit:junit:jar:4.13.1:test
[INFO] |     \- org.hamcrest:hamcrest-core:jar:1.3:test
```
Here you can see that `junit` is pulled as transitive dependency of `com.bazaarvoice.curator:test`. The version of the artifact that is specified in `./parrent/pom.xml` is ignored because `junit` is missing as direct dependency in quality/integration/pom.xml module.

Reply:
version in  [./parent/pom.xml](https://github.com/bazaarvoice/emodb/pull/316/files#diff-b5a06276719e759fe07dfe6f75d781be5f83d2215179d82bdb195ad035348214R69) is `4.13.1`, but i see your point now.
it will be better to get rid of one of the test frameworks(i would leave tesng and remove junit)

Reply:
separate jira ticket has been created to completely migrate to testng